### ISSUE TITLE
Search for typopro-web folder in node_modules

### ISSUE THREAD

**Issue by Reobos** [rse/grunt-typopro#1]
When I install grunt-typopro, the typopro-web npm module dependency is installed in the same top-level node_modules folder. when I try to use grunt with the provided example, it fails to find the **typopro-web** folder (it's looking in the wrong place).

```
$ grunt typopro
Running "typopro" task
Fatal error: no such font: Aleo (expected directory "./node_modules/grunt-typopro/node_modules/typopro-web/web/TypoPRO-Aleo")
```

folder structure is:

```
node_modules/
|   grunt-typopro/
|   |   node_modules (does not contain typopro-web)
|   typopro-web/
|   |   web/
|   |   |   TypoPRO-Aleo
```

If I change in the source file **tasks/grunt-typopro.js (line 34)**
`var typopro_basedir = path.resolve(path.join(__dirname, "..", "node_modules", "typopro-web"));`
to
`var typopro_basedir = path.resolve(path.join(__dirname, "..", "..", "typopro-web"));`
it solves the problem. 

Let me know what details you might want.
Thanks,
Kyle

**Comment by rse** [comment]
Is this already with NPM 3 (where modules are placed towards the top-level)? Because for NPM 2 the typopro-web really is under node_modules and not one level higher.

**Comment by rse** [comment]
Ok, I now resolve the path to "typopro-web" via the "resolve" module and this way it should work under both NPM2 and NPM3. See https://github.com/rse/grunt-typopro/commit/eaff2590188d353a654dfd37b2e82839666d7d1a for more details. I hope this is now fixed with the new release of version 0.9.8. Thanks for reporting the bug.

**Comment by Reobos** [comment]
Great! Thanks for the quick bug fix.
Kyle
